docs/source/l1_to_l2b_e2e.rst

```diff
@@ -0,0 +1,220 @@
+Process L1 to L2b data
+-----------------------
+
+This section demonstrates how process Level 1 (L1) data into Level 2b (L2b) data using the ``corgidrp`` package.
+
+The focus is on explaining the key steps involved in processing data, including setting up calibration files, running the data processing pipeline, and understanding the core steps for transforming raw data into processed output. The test can be run by processing raw L1 data into L2b data using the ``walker.walk_corgidrp`` function. This function is the core of the pipeline and is responsible for the actual data processing. Here is how you can set up and run the test:
+
+Calibration Files
+~~~~~~~~~~~~~~~~~
+
+To process the L1 data, you will need several calibration files. These files are used in the calibration process to adjust the raw data accordingly. The following calibration files are required:
+
+    - Noise Maps: Used to account for noise in the sensor data.
+    - Non-Linearity Calibration: Corrects for any non-linearities in the sensor's response.
+    - Dark Current Calibration: Accounts for any dark current noise in the sensor.
+    - Flat Field Calibration: Used to correct for uneven sensitivity across the detector.
+    - Bad Pixel Maps: Identifies pixels that are defective or not functioning properly.
+
+
+Setup
+~~~~~~
+
+Before running the test, ensure you have the necessary datasets and calibration files. You will need:
+
+1. **L1 Data** - Raw Level 1 data files.
+2. **Cals** - Calibration files required for processing as outlined above. 
+
+The test script is written to process the L1 data, apply the calibration files, and produce the output in the L2b format.
+
+
+.. code-block:: python
+
+    import argparse
+    import os
+    import pytest
+    import numpy as np
+    import astropy.time as time
+    import astropy.io.fits as fits
+    import corgidrp
+    import corgidrp.data as data
+    import corgidrp.mocks as mocks
+    import corgidrp.walker as walker
+    import corgidrp.caldb as caldb
+    import corgidrp.detector as detector
+
+
+Before processing the L1 files, the TVAC FITS headers must be standardized using the ``fix_headers_for_tvac`` function:
+
+Helper function to standardize TVAC FITS headers for pipeline compatibility:
+
+.. code-block:: python
+
+    
+        def fix_headers_for_tvac(list_of_fits):
```

Comment:
the indentation of this line is off. it should be less indented than the rest, since it declares the function.

Reply:
this still needs fixing